aws/rds_cluster_test.go

```diff
@@ -0,0 +1,63 @@
+package aws
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds"
+
+	"github.com/gruntwork-io/cloud-nuke/util"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func createTestRDSCluster(t *testing.T, session *session.Session, name string) {
+	svc := rds.New(session)
+	params := &rds.CreateDBClusterInput{
+		DBClusterIdentifier: awsgo.String(name),
+		Engine:               awsgo.String("aurora-mysql"),
+		MasterUsername:       awsgo.String("gruntwork"),
+		MasterUserPassword:   awsgo.String("password"),
+	}
+
+	_, err := svc.CreateDBCluster(params)
+	require.NoError(t, err)
+
+	//waitUntilRdsCreated(svc, &name)
```

Comment:
Do we need to uncomment that?

Reply:
No, it was remaining from copy + paste 😅